pom.xml

```diff
@@ -67,27 +67,14 @@
             <artifactId>org.apache.sling.testing.sling-mock.core</artifactId>
             <version>${sling-mock.version}</version>
             <scope>provided</scope>
-            <exclusions>
-                <exclusion>
-                    <groupId>org.apache.jackrabbit</groupId>
-                    <artifactId>jackrabbit-api</artifactId>
-                </exclusion>
-            </exclusions>
         </dependency>
         <dependency>
             <groupId>org.apache.sling</groupId>
             <artifactId>org.apache.sling.testing.sling-mock.core</artifactId>
             <version>${sling-mock.version}</version>
             <classifier>tests</classifier>
             <scope>test</scope>
-            <exclusions>
```

Comment:
why were these excludes necessary?

Reply:
jackrabbit-api changed the GAV when moving from JR2 to Oak (https://issues.apache.org/jira/browse/OAK-8339). It is hard to be compatible with both Oak < 1.16 and above for that reason.

Reply:
my goal was that sling-mock itself is fully backwards compatible to AEM 6.4.0, which is 1.8.2. luckily enough it was no problem to use a newer version of sling-mock-oak e.g. with oak 1.16.0 with having both API jackrabbit-api and oak-jackrabbit-api in the classpath. that does no longer work with the version from this PR due to the hard check we currently have.

i think i exclude it here to have the rest run without it, but when using it together with sling-mock or aem-mock its there again.

Reply:
I shaded now also the Jackrabbit API (in its new location). I cannot think of a scenario where this leads to damage.

Reply:
looks good, will to some more tests when this is merged